include/openssl/ocsp.h

```diff
@@ -548,6 +548,13 @@ OPENSSL_EXPORT int OCSP_SINGLERESP_get_ext_count(OCSP_SINGLERESP *sresp);
 OPENSSL_EXPORT X509_EXTENSION *OCSP_SINGLERESP_get_ext(OCSP_SINGLERESP *sresp,
                                                        int loc);
 
+DECLARE_ASN1_FUNCTIONS(OCSP_ONEREQ)
+DECLARE_ASN1_FUNCTIONS(OCSP_RESPDATA)
+DECLARE_ASN1_FUNCTIONS(OCSP_REQINFO)
+DECLARE_ASN1_FUNCTIONS(OCSP_SIGNATURE)
+DECLARE_ASN1_FUNCTIONS(OCSP_RESPBYTES)
+DECLARE_ASN1_FUNCTIONS(OCSP_REVOKEDINFO)
```

Comment:
Just wanted to understand the context and if all of these need to be exported? 

Minor nit: It would be great if we could put these under L118 with all the other `DECLARE_ASN1_FUNCTIONS`s :)

Reply:
I'll move it and I think we can just expose the `OCSP_ONEREQ`. asn1 functions specifically. Context for this change is to be able to support rust-openssl.